### ISSUE TITLE
Dependency on older stdlib

### ISSUE THREAD

**Issue by nicwaller** [ghoneycutt/puppet-module-dnsclient#7]
Does dnsclient depend on having 3.2.x as the exact version of stdlib? That's how the Modulefile is set up right now, so when I install dnsclient it actually downgrades my version of stdlib in order to be compliant.

```
# puppet module install ghoneycutt/dnsclient
Notice: Preparing to install into /etc/puppet/modules ...
Notice: Downloading from https://forge.puppetlabs.com ...
Notice: Installing -- do not interrupt ...
/etc/puppet/modules
âââ¬ ghoneycutt-dnsclient (v3.0.4)
  âââ puppetlabs-stdlib (v4.1.0 -> v3.2.0)
```

Maybe the dependency in the Modulefile should be loosened or removed if it works anyways.

**Comment by tehmaspc** [comment]
@ghoneycutt this issue can be closed right?

**Comment by vchepkov** [comment]
Well, it should actually be updated to support stdlib 6 (and puppet 6)

**Comment by Rewerson** [comment]
@ghoneycutt Any updates on this issue?

**Comment by ghoneycutt** [comment]
Fixed in v4.0.0
